src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -393,8 +405,7 @@ export async function zipCode(modulePath: string) {
     const mavenStatus = mavenFailed ? MetadataResult.Fail : MetadataResult.Pass
     telemetry.codeTransform_jobCreateZipEndTime.emit({
         codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
-        // TODO: A nice to have would be getting the zipUploadSize
-        codeTransformTotalByteSize: 0,
+        codeTransformTotalByteSize: (await fs.promises.stat(tempFilePath)).size,
```

Comment:
Not a blocker: This is the size of the zip folder. I believe for IntelliJ we are logging the byte size pre-zipped.

It will still help us determine overall size of upload/download

Reply:
Aligned offline, will use zip size.